Source/EventFlow.ReadStores.Elasticsearch/Extensions/EventFlowOptionsExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Elasticsearch.Net.ConnectionPool;
+using EventFlow.Aggregates;
+using EventFlow.Configuration.Registrations;
+using EventFlow.Logs;
+using Nest;
+
+namespace EventFlow.ReadStores.Elasticsearch.Extensions
+{
+    public static class EventFlowOptionsExtensions
+    {
+        public static EventFlowOptions UseElasticsearchReadModel<TAggregate, TReadModel>(this EventFlowOptions eventFlowOptions, params string[] indicesToFeed)
```

Comment:
@rasmus The current implementation supports multiple indices - but you have a point - no way to change them at run time. We could take the regex approach as we did in ecgdk - or we could create a service to alter the readmodel->index configuration on runtime.
Or even a combination of both?